### PR TITLE
Martinh/cross chain contracts

### PR DESCRIPTION
**Pull request by martin0995**
### Description

Create Cross-Chain Contracts tutorial page

### Checklist

- [x] **Required** - I have added a label to this PR 🏷️
- [x] **Required** - I have run my changes through Grammarly
- [x] If pages have been moved, I have created redirects in the `wormhole-mkdocs` repo

### CODE REVIEW

File: tutorials/messaging/cross-chain-contracts.md [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+    - **`deployReceiver.js`** - deploys the MessageReceiver contract to Celo
+    - **`sendMessage.js`** - sends a message from Avalanche to Celo
+
+- Configuration files and ABI JSON files for easy deployment and interaction:
+
+    - **`chains.json`** - configuration file that stores key information for the supported TestNets, including the Wormhole relayer addresses, RPC URLs, and chain IDs. You likely won't need to modify this file unless you're working with different networks
+
+### Important Setup Steps
+
+1. **Add Your Private Key** - create a `.env` file in the root of the project and add your private key:
```
Comment (eshaben) [inline tutorials/messaging/cross-chain-contracts.md:135]:
```suggestion
1. **Add your private key** - create a `.env` file in the root of the project and add your private key:
```

Only need to capitalize the first word in all of these list items

Comment (eshaben) [reply tutorials/messaging/cross-chain-contracts.md:135]: Please fix the rest of the list items 🙂

File: tutorials/messaging/cross-chain-contracts.md [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+
+The expected output should be similar to this:
+
+--8<-- "code/tutorials/messaging/cross-chain-contracts/snippet-7.html"
+
+### Deployment Process
+
+Both deployment scripts, `deploySender.js` and `deployReceiver.js`, perform the following key tasks:
+
+1. **Load Configuration and Contract Details** - each script begins by loading the necessary configuration details, such as the network's RPC URL and the contract's ABI and bytecode. This information is essential for deploying the contract to the correct blockchain network.
```
Comment (eshaben) [inline tutorials/messaging/cross-chain-contracts.md:161]:
```suggestion
1. **Load configuration and contract details** - each script begins by loading the necessary configuration details, such as the network's RPC URL and the contract's ABI and bytecode. This information is essential for deploying the contract to the correct blockchain network
```

For all of these list items, I have the same comment as above - you only need to capitalize the first word and then also you don't need punctuation at the end of list items

File: tutorials/messaging/cross-chain-contracts.md [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+
+## Send a Cross-Chain Message
+
+Now that both the sender and receiver contracts are deployed, let's move on to the next exciting step: sending a cross-chain message from Avalanche Fuji to Celo Alfajores.
+
+In this example, we will use the `sendMessage.js` script to transmit a message from the sender contract on Avalanche to the receiver contract on Celo. The script uses [Ethers.js](https://docs.ethers.org/v6/){target=\_blank} to interact with the deployed contracts, calculate the cross-chain cost dynamically, and handle the transaction.
+
+Let's break down the script step by step.
+
+1. Load Configuration Files
```
Comment (eshaben) [inline tutorials/messaging/cross-chain-contracts.md:263]: These can be formatted like in the deployment process section:

```
1. **Load configuration files:**
   1. ....
   2. ....
  
2. **Configure the provider and signer** - the script first reads the chain configurations...
```

Comment (eshaben) [reply tutorials/messaging/cross-chain-contracts.md:263]: Make sure to watch for capitalization

**Review by eshaben — COMMENTED** [review]
Amazing! Just some minor formatting comments to address and then we'll be good to go from my point of view.

@dawnkelly09 since you requested changes, you'll need to give final approval

**Review by dawnkelly09 — APPROVED** [review]
Looks so good! Thank you for all of your work on this.

**Pull request merged**